themes/custom_tmpl/GETTING_STARTED.md

```diff
@@ -0,0 +1,37 @@
+# Configuration
+
+There are three places in the VuFind configuration that you can change to see your custom theme in action.
+
+The first and most straight-forward is by changing `config.ini > [Site] > theme`. This will change the default theme for everyone who comes to your site and is the setting you should change for production.
+
+The two more development-oriented options are below `theme`: `alternate_themes` and `selectable_themes`. They both take comma-separated lists of themes in the format of `name1:theme1,name2:theme2`. Changing the theme via either of these two options sets a cookie that will keep your theme choice over the course of your session.
+
+Setting up themes under `alternate_themes` allows you to add a `ui=name` parameter to the end of your URL to see that theme. For example, the default configuration of `custom:custom_tmpl` would cause `https://vufind.org/?ui=custom` to show the theme described in the directory `custom_tmpl`.
```

Comment:
Would it make sense to only include a zip file of the theme template? That way you must make a copy of the template to use it?